identity-server/clients/src/ConsoleCibaClient/Program.cs

```diff
@@ -23,6 +26,8 @@
 
 await CallServiceAsync(tokenResponse.AccessToken);
 
+// Graceful shutdown
+Environment.Exit(0);
```

Comment:
Todo for Joe - test if this makes the aspire tooling notice the end of the lifetime of this client when run from rider.

Reply:
What I found was that this doesn't help, but what did was updating to the latest early access build of rider.